### PR TITLE
Create CropTopTransformation.java

### PR DESCRIPTION
**Pull request by veresvit**
Same as CropTransformation but the bottom of the image is cropped and the top part is preserved.

### CODE REVIEW

**Comment by wasabeef** [comment]
@veresvit 

Hi. versvit.

Good idea.
but, I think I want to add to CropTransformations.
define the Top and Bottom to enum.

**Comment by veresvit** [comment]
@wasabeef Good idea. Originally I didn't want to interfere with your code.

**Comment by veresvit** [comment]
@wasabeef I implemented the crop type with the enum. Check out the updated branch, please. :)

File: transformations/src/main/java/jp/wasabeef/picasso/transformations/CropTransformation.java [hunk 1]
```diff
@@ -66,4 +75,23 @@ public Bitmap transform(Bitmap source) {
     public String key() {
         return "CropTransformation(width=" + mWidth + ", height=" + mHeight + ")";
```
Comment (wasabeef) [inline transformations/src/main/java/jp/wasabeef/picasso/transformations/CropTransformation.java]: I think should cache for croptype.

```java
return "CropTransformation(width=" + mWidth + ", height=" + mHeight + ", cropType=" + mCropType + ")";
```

**Comment by wasabeef** [comment]
@veresvit 
Good Job

I will release on the weekend. :+1:

**Comment by wasabeef** [comment]
@veresvit 

1.0.3 has been released.

**Comment by veresvit** [comment]
Fast work @wasabeef ! :+1:

**Pull request merged**